code/script.js

```diff
@@ -1,47 +1,150 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const form = document.getElementById('name-form')
+const inputText = document.getElementById('name-input')
+const inputWrapper = document.getElementById('input-wrapper')
+const messageAudio = new Audio ("./assets/button_click.mp3");
 
 // Global variables, if you need any, declared here
 
 // Functions declared here
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
+  console.log ('hi there')
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/girl.png" alt="User" />  
       </section>
     `
   } else if (sender === 'bot') {
+    console.log ('what do you want?')
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/operator.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
+
+  //Function that triggers click sound
+  messageAudio.currentTime = 0
+  messageAudio.play()
+
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+//Startup greeting after load (timeout 1000)
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Good day Sir/Ms, please type your name`, 'bot')
 }
 
-// Set up your eventlisteners here
+// First function handle name input
+const handleNameInput =  (event) => {
+  event.preventDefault()
+  const name = inputText.value
+  showMessage(`My name is ${name}`, 'user')
+  inputText.value = ''
+  setTimeout(() => destination(), 1000)
+}
+
+//Second function choose of destination
+const destination = () => {
+  showMessage(`Choose your destination here`, 'bot')
+  inputWrapper.innerHTML = `
+    <button id="gotBtn">Gothenburg</button>
+    <button id="mmxBtn">Malmö</button>
+    <button id="stoBtn">Stockholm</button>`
 
+  document
+    .getElementById('gotBtn')
+    .addEventListener('click', () => { showMessage('Gothenburg', 'user')
+    setTimeout(() => showMessage('Gothenburg is a nice place, choose your vehicle', 'bot'), 1000)
+    inputWrapper.innerHTML = ''
+      setTimeout(vehicle, 2000)
+  })
+
+  document
+    .getElementById('mmxBtn')
+    .addEventListener('click', () => { showMessage('Malmö', 'user')
+    setTimeout(() => showMessage('Malmö is a nice place, choose your vehicle', 'bot'), 1000)
+    inputWrapper.innerHTML = ''
+    setTimeout(vehicle, 2000)
+  })
```

Comment:
Smart thing to put the whole code inside the document! It makes it easy to follow the steps in the code